drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -20,6 +20,8 @@
 #define I2C_STANDARD_PORT_COUNT 3
 /* Default PLL frequency. */
 #define PLL_CLOCK 48000000
+/* I2C Standard mode speed option: 50 KHz */
+#define I2C_BITRATE_SM_OPT_50KHZ 50000
```

Comment:
Is there a way to avoid this specific freq, like a little quick mapping of bitrate to closest clock settings that doesn't involve having a hardcoded macro here?